rotkehlchen/chain/ethereum/modules/uniswap/v3/utils.py

```diff
@@ -265,15 +278,22 @@ def uniswap_v3_lp_token_balances(
             tokens_b,
         ):
             liquidity_in_pool = _entry[0].decode(_entry[1][1], 'liquidity')[0]
-            total_tokens_in_pools.append(
-                calculate_total_amounts_of_tokens(
-                    liquidity=liquidity_in_pool,
-                    tick=_entry[3][1],
-                    fee=_entry[2][4],
-                    decimal_0=_entry[4]['decimals'],
-                    decimal_1=_entry[5]['decimals'],
-                ),
-            )
+            try:
+                total_tokens_in_pools.append(
+                    calculate_total_amounts_of_tokens(
+                        liquidity=liquidity_in_pool,
+                        tick=_entry[3][1],
+                        fee=_entry[2][4],
+                        decimal_0=_entry[4]['decimals'],
+                        decimal_1=_entry[5]['decimals'],
+                    ),
+                )
+            except UnrecognizedFeeTierException as e:
+                error_msg = f'Error calculating total amount of tokens in pool due to: {str(e)}'
+                log.error(error_msg)
```

Comment:
I will merge since this is no reason to run the tests again but Isaac, remember if you use `msg_aggregator.add_error(X)` then there is no reason to also do `log.error(X)` since the first one also includes the latter inside.